dotcom-rendering/src/web/components/ArticleMeta.tsx

```diff
@@ -407,10 +415,17 @@ export const ArticleMeta = ({
 								className="meta-number"
 								id="share-count-root"
 							/>
-							<div
-								className="meta-number"
-								id="comment-count-root"
-							/>
+							<div className="meta-number">
+								{isCommentable && (
+									<Island clientOnly={true}>
+										<CommentCount
+											discussionApiUrl={discussionApiUrl}
+											shortUrlId={shortUrlId}
+											format={format}
+										/>
+									</Island>
+								)}
+							</div>
```

Comment:
This is the main change in this PR. I've removed the code in App.tsx (well, actually I removed it from `Discussion`) which was adding this component and it now lives in its own Island